jobs/JREGIONAL_MAKE_LBCS

```diff
@@ -64,6 +74,7 @@ mkdir_vrfy -p "${lbcs_dir}"
 #-----------------------------------------------------------------------
 #
 $SCRIPTSDIR/exregional_make_lbcs.sh \
+  time_offset_hrs="${time_offset_hrs}" \
```

Comment:
The variable `EXTRN_MDL_LBCS_OFFSET_HRS` will be available to the script as-is. No need to pass it in the arguments as a different name.

Reply:
removed time_offset_hrs